srcalc/src/main/webapp/css/srcalc.css

```diff
@@ -337,6 +337,18 @@ a.openProcedureSelect {
 	color: #aaa;
 }
 
+/* === Admin Page-specific Styles === */
+
+.editObjectLink {
+	text-decoration: none;
+}
+
+ul#newVariableLinks {
+	/* Display these in a row, not as a bullet list. */
+	list-style-type: none;
+	padding: 0;
```

Comment:
I think this would be okay in a row, but I feel like sufficient padding between links would be better than having an ellipsis after each link.